content/docs/reference/spec/migration/platform-api.md

```diff
@@ -0,0 +1,42 @@
++++
+title="Platform API 0.3 -> 0.4"
```

Comment:
The title of this doesn't match the file name. I think we'd like to prepare ourselves for having multiple migration guides for platform API hence I would recommend that the file name be changed to `platform-api-0.3-0.4.md`